extensions/cornerstone/src/initMeasurementService.ts

```diff
@@ -12,6 +12,7 @@ import {
   triggerAnnotationRenderForViewportIds,
 } from '@cornerstonejs/tools/utilities';
 import getActiveViewportEnabledElement from './utils/getActiveViewportEnabledElement';
+import { servicesManager } from 'platform/app/src/App';
```

Comment:
You can't import things this way - you need to import from a package itself, as the extensions should never try to directly reference a source file in the platform.